### PR TITLE
MS16887: Fixing Handshake Spam Crash

### PR DESCRIPTION
**Pull request by wayne-chen**
- Fix for MS[#16887](https://highfidelity.manuscript.com/f/cases/16887)

**TEST PLAN**
User 1 - Desktop (or HMD) User
User 2 - HMD User

User 1 and 2: Launch Interface
User 1 and 2: Load into domain with another person
User 1: Log in
User 1: Hold down "x" (or "Grip" in HMD mode)
User 2: Hold out your hand and press "Grip" rapidly (Attempt this for at least 45s)
User 2 should not crash

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11477/HighFidelity-Beta-PR13664-5472d17.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11477/HighFidelity-Beta-PR13664-5472d17.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11477/HighFidelity-Beta-Interface-PR13664-5472d17.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11477/HighFidelity-Beta-PR13664-5472d17.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11477/HighFidelity-Beta-Interface-PR13664-5472d17.exe

File: scripts/system/makeUserConnection.js [hunk 1]
```diff
@@ -920,10 +918,6 @@
     // connect updateVisualization to update frequently
     Script.update.connect(updateVisualization);
 
-    // load the sounds when the script loads
-    handshakeSound = SoundCache.getSound(HANDSHAKE_SOUND_URL);
-    successfulHandshakeSound = SoundCache.getSound(SUCCESSFUL_HANDSHAKE_SOUND_URL);
```
Comment (ctrlaltdavid) [inline scripts/system/makeUserConnection.js]: The sounds are loaded at start-up so that the play immediately, first time they're played.

**Comment by qlistephens95** [comment]
Currently Testing.

**Comment by qlistephens95** [comment]
FAIL @conklin94122 . While spamming the handshake button either via desktop or via HMD the interface still crashes. Log attached.
Steps to reproduce:
1: Launch interface assuming user(s) have installed correct build.
2: make sure the two users are in the same domain.
3: User 1 attempt to spam handshake button while User 2 attempts to receive handshake.
4: While spamming the handshake button the interface crashes.
[hifi-log.txt](https://github.com/highfidelity/hifi/files/2219877/hifi-log.txt)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11503/HighFidelity-Beta-PR13664-020d908.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11503/HighFidelity-Beta-PR13664-020d908.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11503/HighFidelity-Beta-Interface-PR13664-020d908.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11503/HighFidelity-Beta-PR13664-020d908.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11503/HighFidelity-Beta-Interface-PR13664-020d908.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12346/HighFidelity-Beta-PR13664-a90f01e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12346/HighFidelity-Beta-Interface-PR13664-a90f01e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12346/HighFidelity-Beta-PR13664-a90f01e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12346/HighFidelity-Beta-Interface-PR13664-a90f01e.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12356/HighFidelity-Beta-PR13664-f110a67.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12356/HighFidelity-Beta-PR13664-f110a67.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12356/HighFidelity-Beta-Interface-PR13664-f110a67.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12364/HighFidelity-Beta-PR13664-f9a3db1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12364/HighFidelity-Beta-PR13664-f9a3db1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12364/HighFidelity-Beta-Interface-PR13664-f9a3db1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12364/HighFidelity-Beta-PR13664-f9a3db1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12364/HighFidelity-Beta-Interface-PR13664-f9a3db1.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13086/HighFidelity-Beta-PR13664-f67ba50.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13203/HighFidelity-Beta-PR13664-d314259.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13262/HighFidelity-Beta-PR13664-4094000.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13275/HighFidelity-Beta-PR13664-58ef98b.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13275/HighFidelity-Beta-PR13664-58ef98b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13275/HighFidelity-Beta-Interface-PR13664-58ef98b.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13275/HighFidelity-Beta-PR13664-58ef98b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13275/HighFidelity-Beta-Interface-PR13664-58ef98b.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13310/HighFidelity-Beta-PR13664-f9c6464.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13312/HighFidelity-Beta-PR13664-5a8daf2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13312/HighFidelity-Beta-Interface-PR13664-5a8daf2.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13419/HighFidelity-Beta-PR13664-3562abb.apk

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13557/HighFidelity-Beta-PR13664-3562abb.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13557/HighFidelity-Beta-PR13664-3562abb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13557/HighFidelity-Beta-Interface-PR13664-3562abb.exe

**Comment by qlistephens95** [comment]
Currently testing

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13585/HighFidelity-Beta-PR13664-1228604.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13585/HighFidelity-Beta-PR13664-1228604.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13585/HighFidelity-Beta-Interface-PR13664-1228604.exe

**Comment by shanzzam** [comment]
Manuscript ticket has pushed to RC76. Setting Milestone to reflect that.

**Comment by shanzzam** [comment]
Cross-posting from manuscript ticket, this PR hasn't seen any movement in 20 days and has a DNM label. It's within the project scope of 76 but I'm reluctant to approve anything with a DNM label.

**Pull request closed without merging**